modules/browser/BrowserCapabilities.js

```diff
@@ -170,7 +157,6 @@ export default class BrowserCapabilities extends BrowserDetection {
      */
     supportsRtpStatistics() {
         return this.isChromiumBased()
-            || this.isEdge()
```

Comment:
Is there a browser where this is not supported now? If not, maybe let's drop the function completely.

Reply:
sure, we could drop it.

Reply:
👍